src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SampleData.java

```diff
@@ -104,12 +100,10 @@
 
     }
 
-    public static String fromFile(String file) {
+    public static byte[] fileContentAsBytes(String file) {
```

Comment:
i wonder if this should just be `exampleJsonAsBytes` sample data shouldn't be as generic as this imo, this is more of a util

Reply:
I agree that it is more of a util, but it's certainly more generic than `exampleJsonAsBytes`

Reply:
is it though? you only needed to make one change in tests

Reply:
I am in favour of more generic methods generally. Have a look now